backend/.dockerignore

```diff
@@ -0,0 +1,9 @@
+## Ignore all files
+*
+## Do not ignore
+!src/
+!tsconfig.json
+!package.json
+!package-lock.json
+!config.json
```

Comment:
Config is in `.gitignore` and should not be included into the image.
You should include `tsconfig.json` though, otherwise the build will not match what we expect it to be.